decode/radio_frame.js

```diff
@@ -1,45 +1,43 @@
 var EthernetAddr = require('./ethernet_addr');
 var LogicalLinkControl = require('./llc_packet');
-
+var RadioBeaconFrame = require('./radio_beacon_frame');
 function RadioFrame() {
 
 }
 
 RadioFrame.prototype.decode = function (raw_packet, offset) {
     var ret = {};
-
-    ret.frameControl = raw_packet.readUInt16BE(offset, true);
-    offset += 2;
+    ret.frameControl = raw_packet.readUInt16LE(offset, true); offset += 2;
+    ret.version = ret.frameControl & 0x0003;
     ret.type = (ret.frameControl >> 2) & 0x0003;
     ret.subType = (ret.frameControl >> 4) & 0x000f;
-    ret.flags = (ret.frameControl >> 8) & 0xff;
+    var flags = (ret.frameControl >> 8) & 0xff;
+    ret.flags = { raw:flags };
```

Comment:
What do you think about putting all the IEEE802.11 related types in a subfolder?